README.md

```diff
@@ -1,5 +1,8 @@
 # PromptGuard
-PromptGuard enables applications to leverage the power of language models without compromising user privacy. This repo contains the PromptGuard Python library, which provides a simple interface for interacting with the PromptGuard API. More information about PromptGuard can be found in the [documentation](https://promptguard.readthedocs.io/).
+PromptGuard enables applications to leverage the power of language models while preserving user privacy. This repo contains the PromptGuard Python library, which provides a simple interface for interacting with the PromptGuard API. More information about PromptGuard can be found in the [documentation](https://promptguard.readthedocs.io/).
+
+**API Stability:** This package is still in development. As such, its API may
```

Comment:
We should avoid overloading the term API -- we use it above to indicate the actual service. What if we call this the Python API, and the PromptGuard API above the PromptGuard Service?